app/templates/serviceLearning/slcProposal.html

```diff
@@ -1,4 +1,11 @@
-<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
+ <h2 class="text-center pt-4 pb-2">Proposal For Service-Learning Course</h2>
+
+      <h3 class="text-center pb-4">Proposal Status:
```

Comment:
"Proposal Status" is redundant. "Status" should be fine.

Reply:
Changed.